frontend/test/__support__/e2e/helpers/e2e-dashboard-helpers.js

```diff
@@ -50,3 +51,93 @@ export function addTextBox(string, options = {}) {
     "You can use Markdown here, and include variables {{like_this}}",
   ).type(string, options);
 }
+
+export function createStructuredQuestion(products_table_id, product_category) {
```

Comment:
The old way was to use `PRODUCTS_ID` and `PRODUCTS.CATEGORY` (from the sample DB JSON file) directly, but there's a load order issue where the sample DB file hasn't been created yet when loading the helper files. I'm not thrilled that I had to parameterize these three functions (and I'm extra not-thrilled that they each have a different signature), but I didn't see a way around it. I'm certainly open to suggestions from those more skilled in the wiles of Cypress.

Reply:
How could this happen? Should the test be fixed instead so the helper won't be called early?